Tests/Debug/TraceableDomainEventDispatcherTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Biig\Component\Domain\Tests\Debug;
+
+use Biig\Component\Domain\Debug\TraceableDomainEventDispatcher;
+use Biig\Component\Domain\Event\DomainEvent;
+use Biig\Component\Domain\Event\DomainEventDispatcher;
+use Biig\Component\Domain\Exception\InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class TraceableDomainEventDispatcherTest extends TestCase
```

Comment:
Many tests are missing isn't it?

Reply:
You got the `DomainEventDispatcherInterface` interface that you can use for mocking

Reply:
OK I did that. Set as distinct commit as it alters how TraceableEventDispatcher is built.